app/frontend/src/views/CDOGS.vue

```diff
@@ -34,3 +54,12 @@ export default {
   }
 };
 </script>
+
+<!-- Vuetify uses !important internally, so we have to override ALL of them same way -->
+<style lang="scss" scoped>
+.alert {
+  color: #6c4a00 !important;
```

Comment:
not a big deal and you prob know but good to avoid using !important if possible.